### PR TITLE
More Galar NPCs (Hints & Darkest Day Questline)

### PR DESCRIPTION
**Pull request by CorgiOnNeptune**
I had some written that conflicted a bit with Qwerty's PR. The only one I left in that still conflicts a bit is the Hiker in Hammerlocke for the Cramorant. The cook isn't really needed also, but I thought it was funny so I kept it in. If we feel it just doesn't serve a purpose to keep in, that's fine too.

NPC talking about Farfetch'd's leek is commented out for now till that evo is ready so I have a better idea where to place it.

### CODE REVIEW

File: src/scripts/towns/TownList.ts [hunk 1]
```diff
@@ -4390,40 +4394,140 @@ const FreezingtonShop = new Shop([
...
-    'Our Marnie\'s Morpeko is so cute, don\'tcha think?',
-    'When it\'s hungry, though, it gets so angry it changes forms',
+    'Our Marnie’s Morpeko is so cute, don’tcha think?',
+    'When it’s hungry, though, it gets so angry it changes forms.',
     'If you want to catch it in that form, you would probably be best trying to attract it with a berry that electric Pokémon like.',
 ], {image: 'assets/images/trainers/Team Yell Grunts.png'});
+
+/* To be added when held item evos are ready.
+// TODO: Add to town near a dungeon/shop that has the Leek available.
+const FarfetchdArtist = new NPC('Artist Duncan', [
```
Comment (Jaaslet) [inline src/scripts/towns/TownList.ts:4515]: Leek will be added as a drop item for Galarian Farfetch'd, so you can just finish this NPC

File: src/scripts/towns/TownList.ts [hunk 2]
```diff
@@ -4723,11 +4834,11 @@ const PeonyComplete = new NPC ('Peony', [
 const ProfMagnolia = new ProfNPC('Prof. Magnolia',
     GameConstants.Region.galar,
     'TODO: Add text before Galar is released',
-    'TODO: Add text before Galar is released');
-
+    'TODO: Add text before Galar is released',
```
Comment (Jaaslet) [inline src/scripts/towns/TownList.ts:4837]: Any chance you can add these texts? Can't remember what they need to say, but you can look at the other ProfNPC's

File: src/scripts/temporaryBattle/TemporaryBattleList.ts [hunk 3]
```diff
@@ -231,7 +231,7 @@ TemporaryBattleList['Suicune 6'] = new TemporaryBattle(
 TemporaryBattleList['Spiky-eared Pichu'] = new TemporaryBattle(
     'Spiky-eared Pichu',
     [new GymPokemon('Spiky-eared Pichu', 3178500, 20)],
-    '',
+    '<b><i>You caught the Spiky-eared Pichu!</i></b>',
```
Comment (Jaaslet) [inline src/scripts/temporaryBattle/TemporaryBattleList.ts:234]: It's defeated message will now show up (before the window just went away).
So we now need to add an image for the pichu, so the modal has something to show

**Review by Jaaslet — CHANGES_REQUESTED** [review]
Looks good! I don't know if there are too many lore-NPCs. I will hopefully get a feeling, next time i play through Galar. But i think 2-4 per region is what we want. Don't know how many you added.
And also some more comments:

File: src/scripts/towns/TownList.ts (hunk 2 above)
Comment (CorgiOnNeptune) [reply src/scripts/towns/TownList.ts]: Prof Kukui was also missing this text for Alola. I added them in, with the next region line for Magnolia noting that there is no ETA when the next region is available, with a commented out line to be changed to when it is.

File: src/scripts/temporaryBattle/TemporaryBattleList.ts (hunk 3 above)
Comment (CorgiOnNeptune) [reply src/scripts/temporaryBattle/TemporaryBattleList.ts:233]: I used the directory to not have to add another image in the temporaryBattle folder. If we like this solution I can clean up some of the other potentially unnecessary image files in the temporaryBattle folder to save up a tiny bit of space using the same method in a different PR.

**Review by Jaaslet — APPROVED** [review]
Looks good. Thanks for the PR

**Pull request merged**